src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/equalsavoidnull/InputEqualsAvoidNullEnhancedInstanceof.java

```diff
@@ -12,20 +12,21 @@
     public InputEqualsAvoidNullEnhancedInstanceof(String str) {
         if (str instanceof String myString) {
             System.out.println("Mystring!!");
-            boolean myBool = myString.equals("MyString!!"); // violation
+            boolean myBool = myString.equals("MyString!!"); // violation 'String literal expressions should be on the left side of an equals comparison.'
             boolean myOtherBool = myString.equals(str); // ok
         } else if (str instanceof String oneMoreString) {
-            if(oneMoreString.equals("test")) { // violation
+            if(oneMoreString.equals("test")) { // violation 'String literal expressions should be on the left side of an equals comparison.'
                 System.out.println("Test!!");
             }
         }
     }
 
     void foo (Object string) {
         if(string instanceof String string1) {
-            final boolean myBool = string1.equals("test1") // violation
-                    || string1.equals("test1.5") // violation
-                    || string1.equals("test2") || string1.equals("test3"); // violation 2x
+            final boolean myBool = string1.equals("test1") // violation 'String literal expressions should be on the left side of an equals comparison.'
```

Comment:
long lines is attention killer of reviewer or future reader of this code.
It is not acceptable.
Please shorten a message, we do not need full/whole message, we need some short part of message to quickly understand violation.

Reply:
done